equinix/data_source_fabric_service_profile_acc_test.go

```diff
@@ -0,0 +1,105 @@
+package equinix_test
+
+import (
+	"fmt"
+	"github.com/equinix/terraform-provider-equinix/internal/acceptance"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"regexp"
+	"testing"
+)
+
+func testAccFabricReadServiceProfileConfig(uuid string) string {
```

Comment:
We don't want to rely on Service Profiles being created already for our Data_Source Reads. Go ahead and create one in the same config as the data_sources, and have them use it with `equinix_fabric_service_profile.resource_name.id` or `equinix_fabric_service_profile.resource_name.name`

Then our tests won't have the dependency on a Service Profile that could be deleted from our UAT environment and break our tests.

Reply:
done